resources/qml/Avatar.qml

```diff
@@ -0,0 +1,51 @@
+import QtQuick 2.6
+import QtGraphicalEffects 1.0
+import Qt.labs.settings 1.0
+
+Rectangle {
+	id: avatar
+	width: 48
+	height: 48
+	radius: settings.avatar_circles ? height/2 : 3
```

Comment:
If the user is using squares, should the avatars have any rounding applied at all?

Reply:
That's what the old timeline did.

Reply:
Yeah... just wondering if it's appropriate.

Reply:
I think it is. The rounding just cuts 3 pixels in every corner or so, but it makes the avatars look much nicer, as otherwise they have pretty hard corners. So almost no detail is lost and they look square, but the fit a bit better. If you look carefully, pretty much every application does something like that, i.e. most application icons (KDEs Konsole for example) or the Github Avatars.